cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/common/DefaultStageConfigurer.java

```diff
@@ -75,6 +77,12 @@ public void setOutputSchemas(Map<String, Schema> outputSchemas) {
     return inputSchemas;
   }
 
+  @Nullable
```

Comment:
Will it ever be null?? For stage that doesn't have input (source?), it should be just an empty list, not `null`.

Reply:
removed Nullable annotation.